### PR TITLE
Teams pricing update

### PR DESCRIPTION
**Pull request by DMarby**
Changes Teams pricing to $12 a month / $120 a year, except for if you currently have an active Teams subscription, then you are allowed to change the amount of seats and retain the old pricing.
Once you cancel your subscription or downgrade to Plus, you are put on the new pricing plan if you resubscribe.

Worth nothing that this PR contains the absolutely smallest number of changes needed to do this, as this code is very brittle, so I did not include any cleanup or such.

image
image

### CODE REVIEW

File: src/pages/app/subscribe.js [hunk 1]
```diff
@@ -9,13 +9,17 @@ const planTypePlus = 'plus';
...
 const planIdMap = {
-  'plus-monthly-1': [planTypePlus, planCycleMonthly, 1],
-  'plus-yearly-1': [planTypePlus, planCycleYearly, 1],
-  'team-monthly-1': [planTypeTeam, planCycleMonthly, 5],
-  'team-yearly-1': [planTypeTeam, planCycleYearly, 5]
+  'plus-monthly-1': [planTypePlus, planCycleMonthly],
+  'plus-yearly-1': [planTypePlus, planCycleYearly],
+  'team-monthly-1': [planTypeTeam, planCycleMonthly],
+  'team-yearly-1': [planTypeTeam, planCycleYearly],
+  'team-monthly-2': [planTypeTeam, planCycleMonthly],
```
Comment (DMarby) [inline src/pages/app/subscribe.js:21]: These are just used for mapping a users existing subscription to the type/billing interval.
The third argument was unused, so I removed it, presumably it was used for default quantity at some point in time.

File: src/pages/app/subscribe.js [hunk 2]
```diff
@@ -31,15 +35,21 @@ class Subscribe extends React.Component {
 
     let planDescription;
     if (window.location.hash === '#teams') {
-      planDescription = planIdMap['team-monthly-1'];
+      planDescription = planIdMap['team-monthly-2'];
```
Comment (DMarby) [inline src/pages/app/subscribe.js:38]: These doesn't strictly need to change, but might as well.

**Review by develohpanda — APPROVED** [review]
🔥

File: src/pages/app/subscribe.js (hunk 2 above)
Comment (develohpanda) [reply src/pages/app/subscribe.js:38]: Seems reasonable

**Review by nijikokun — APPROVED** [review]

**Pull request merged**